modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -0,0 +1,280 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-account.adoc
+
+[id="minimum-required-azure-permissions_{context}"]
```

Comment:
This ID is the same as the one in modules/minimum-required-permissions-upi-azure.adoc. Please change both module IDs so that they match the corresponding filename. Ideally, they would also match the module title, but having them match the filename is more important.

This one should be: `[id="minimum-required-permissions-ipi-azure_{context}"]`